Dockerfile

```
@@ -0,0 +1,34 @@
+FROM php:7.4-fpm
+
+# Arguments defined in docker-compose.yml
+ARG user
+ARG uid
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+    git \
+    curl \
+    libpng-dev \
+    libonig-dev \
+    libxml2-dev \
+    zip \
+    unzip
+
+# Clear cache
+RUN apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# Install PHP extensions
+RUN docker-php-ext-install pdo_mysql mbstring exif pcntl bcmath gd
+
+# Get latest Composer
+COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
+
+# Create system user to run Composer and Artisan Commands
+RUN useradd -G www-data,root -u $uid -d /home/$user $user
```

```suggestion
RUN useradd -b /var -UG www-data -s /usr/sbin/nologin --system $user
```

So you think just this is enough @dshoreman?

---

Yep, should do it. Application code gets mapped to `/var/servidor` in the guest (or real server), so we set that as the home directory. That way, Composer dumps caches etc to `/var/servidor/.composer` to solve permission issues

---

@dshoreman should we switch volume mount from `./:/var/www` to `./:/var/servidor`?